design/templates/solution_design.md

```diff
@@ -52,59 +116,80 @@
+| **Scenario** | **Log message** |
+|--------------|-----------------|
+|              |                 |
+|              |                 |
+
+## Documentation
+[//]: # "Add notes on what should be documented in this solution. Elaborate on where this should be documented. If the change is in open-source projects, we may need to update the docs in github too. If it's in Conjur and/or DAP, mention which products are affected by it"
+
+## Security
+[//]: # "Are there any security issues with your solution? Even if you mentioned them somewhere in the doc it may be convenient for the security architect review to have them centralized here"
 
-### Designs 
-[//]: # "Note and estimate the tasks which are related to designs which needed for the solution"
+| **Security Issue** | **Description** | **Resolution** |
+|--------------------|-----------------|----------------|
+|                    |                 |                |
 
-### Spikes and researches
-[//]: # "Note and estimate the tasks which are related to spikes and researches which needed for the solution"
+## Audit 
+[//]: # "Does this solution require adding audit logs? Does it affect existing audit logs?"
 
-### Implementation 
-[//]: # "Note and estimate the tasks which are related to the solution's code implementation"
+| **Name (ID)** | **Description** | **Issued On** |
+|---------------|-----------------|---------------|
+|               |                 |               |
 
-### Testing 
-[//]: # "Note and estimate the tasks which are related to writing/performing tests [automation, manual tests, etc.]"
+## Open Questions
+[//]: # "Add any question that is still open. It makes it easier for the reader to have the open questions accumulated here instead of them being acattered along the doc"
 
-### Security 
-[//]: # "Note and estimate the tasks which are related to security researches and implementations"
+## Definition of Done
 
-### Docs 
-[//]: # "Note and estimate the tasks which are related to writing the docs"
+- Solution designed is approved 
+- Test plan is reviewed
+- Acceptance criteria have been met
+- Tests are implemented according to test plan 
+- The behaviour is documented in DAP and in Conjur 
+- All relevant components are released
 
-### Demo 
-[//]: # "Note and estimate the tasks which are related to creating a demo that demonstrates the solution"
+## Solution Review
+[//]: # "Relevant personas can indicate their design approval by approving the pull request"
 
-### Left overs and refactoring 
-[//]: # "Note and estimate the task that needs to be done as part of this effort but will not be delivered in the initial release and will be developed later"
\ No newline at end of file
+| **Persona**        | **Name** |
+|--------------------|----------|
+| Team leader        |          |
+| Product owner      |          |
+| System architect   |          |
+| Security architect |          |
+| QA architect       |          |
```

Comment:
```suggestion

- [ ] Engineering manager - _name_
- [ ] Product owner - _name_
- [ ] Software architect - _name_
- [ ] Security architect - _name_
- [ ] QA architect - _name_

```
This way, the relevant people can check the box

Reply:
I proposed having PR approvals serve as sign off - changing the MD content to check the box is not a great UX